templates/probot/files/README.md

```diff
@@ -0,0 +1,91 @@
+# Probot GitHub Apps for Platform.sh
+
+<p align="center">
+<a href="https://console.platform.sh/projects/create-project?template=https://raw.githubusercontent.com/platformsh/template-builder/master/templates/probot/.platform.template.yaml&utm_content=probot&utm_source=github&utm_medium=button&utm_campaign=deploy_on_platform">
+    <img src="https://platform.sh/images/deploy/lg-blue.svg" alt="Deploy on Platform.sh" width="180px" />
+</a>
+</p>
+
+This template builds a simple GitHub App using [Probot](https://github.com/probot/probot).
+
+Probot is a framework for building GitHub Apps in Node.js.
+
+## Services
+
+* Node.js 12
+
+## Post-install
+
+1. **Register the App on GitHub**
+
+   When the app has deployed, you will need to first register the app with GitHub by clicking the **Register GitHub App** button on the homepage. Name the application anything you like, then click the **Create GitHub App** button. This will automatically save your `WEBHOOK_SECRET`, `PRIVATE_KEY`, and `APP_ID` variables to a `.env` file in your Platform.sh environment.
```

Comment:
Isn't that file read-only?  Or is it not in the app root?

Reply:
Ah, I see the code in the build hook below.  That should be clarified here that it's a symlink to a writeable file.